pkg/db/db.go

```diff
@@ -171,7 +178,7 @@ func (c *Client) updateDownloadedAt(dst string) error {
 	return nil
 }
 
-func (c *Client) populateOCIArtifact() error {
+func (c *Client) populateOCIArtifact(dbRepository string) error {
```

Comment:
We have `dbRepository` as a member of `Client`. Do we need to pass it as an argument? We can access `c.dbRepository` in this method.